src/tray.js

```diff
@@ -16,14 +16,16 @@ const { CONFIG_KEY: PUBSUB_KEY } = require('./enable-pubsub')
 const { CONFIG_KEY: NAMESYS_PUBSUB_KEY } = require('./enable-namesys-pubsub')
 const { CONFIG_KEY: AUTO_GC_KEY } = require('./automatic-gc')
 const { CONFIG_KEY: AUTO_LAUNCH_WEBUI_KEY } = require('./webui')
+const { CONFIG_KEY: ASK_OPENING_IPFS_URIS } = require('./protocol-handlers')
 
 const CONFIG_KEYS = [
   AUTO_LAUNCH_KEY,
   AUTO_LAUNCH_WEBUI_KEY,
   AUTO_GC_KEY,
   SCREENSHOT_KEY,
   PUBSUB_KEY,
-  NAMESYS_PUBSUB_KEY
+  NAMESYS_PUBSUB_KEY,
+  ASK_OPENING_IPFS_URIS
```

Comment:
@lidel @achingbrain This is why we need to use trailing commas. How can I go about updating the styling across our repos so that we can allow/require trailing commas in our code? These diffs don't need to be this noisy. git blaming is also a PITA when we try to find out who originally added NAMESYS_PUBSUB_KEY, or whatever the non-trailing-comma-having previous line ends up being in PRs.